class/Defaults/Trigger/Media/MediaTrigger.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Media trigger abstract
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Defaults\Trigger\Media;
+
+use BracketSpace\Notification\Abstracts;
+use BracketSpace\Notification\Defaults\MergeTag;
+/**
+ * Post trigger class
+ */
+abstract class MediaTrigger extends Abstracts\Trigger {
+	/**
```

Comment:
Please make some room class body, here on the top and on the bottom as well

Reply:
I cannot make a spacing on the top "There must be no blank lines after the class comment"

Reply:
Noo I meant:
```
class Asd {
    /** Your code **/
}
```
->
```
class Asd {

    /** Your code **/

}
```

Reply:
And this is exactly what you've done (beside the space on the bottom), but let's leave it like this.

Reply:
i cannot add spacing on the bottom, my phpcs was : 

>  Whitespace found at end of line